### ISSUE TITLE
How to train without learned model?

### ISSUE THREAD

**Issue by kimna4** [ljanyst/ssd-tensorflow#19]
Hi, I'am reading your SSD code. It is really good for me. Thank you.

I hope to train the SSD without learned model to see decreasing losses.
But, there are no parameters or boolean variables for switching train mode.

Is there any simple way to train without any model?

Thank you.

**Comment by ljanyst** [comment]
I am not quite sure what you're asking for. If you mean a custom data set, then yes - you can create a source file for whatever dataset you want. If you mean not using a pretrained VGG network, then no - you need weights for VGG16 pre-trained on Imagenet.

**Comment by kimna4** [comment]
My question is not using a pretrained VGG network.
Thank you. I will modify the source code for my research.
